skimage/io/_io.py

```diff
@@ -130,6 +130,9 @@ def imsave(fname, arr, plugin=None, **plugin_args):
             plugin = 'tifffile'
     if is_low_contrast(arr):
         warn('%s is a low contrast image' % fname)
+    if (type(arr) == np.ndarray and arr.dtype == bool) or \
+        any(isinstance(ele, bool) for ele in arr):
```

Comment:
@alexandrejaguar the function you are looking for is `input_array = np.asanyarray(input_array)`. =)